decidim-admin/app/assets/javascripts/decidim/admin/sort_steps.js.es6

```diff
@@ -6,6 +6,13 @@
 // should have a `data-sort-url` attribute with the URL where the data should
 // be posted to.
 $(() => {
```

Comment:
This code should be move to `application.js.es6`. It's better to handle events in a single file so you can include the `sortSteps()` call for both document ready and turbolinks ready events.